iroh/src/magicsock.rs

```diff
@@ -563,16 +566,16 @@ impl MagicSock {
                         // at any time so these errors should be treated as transient and
                         // are just timeouts.  Hence we opt for returning Ok.  See
                         // test_try_send_no_udp_addr_or_relay_url to explore this further.
-                        error!(
+                        debug!(
                             node = %node_id.fmt_short(),
-                            "no UDP or relay paths available for node",
+                            "no UDP or relay paths available for node, voiding transmit",
                         );
                     }
                     Ok(())
                 }
             }
             None => {
-                error!(%dest, "no NodeState for mapped address");
+                debug!(%dest, "no NodeState for mapped address, voiding transmit");
```

Comment:
How are we sending to something we have no NodeState for?  AFAIK that's genuinely bad and a bug we should fix.  We should have a QuicMappedAddr with NodeId for everything that makes it this far should we not?

Reply:
I don't think I've actually hit that in practice yet. So I'm fine with making this `error` again, or even panicking or sth like that.

That said, there's a comment below this talking about how this is fine, so I thought it's fine?

Reply:
Yeah, the socket and quic might be able to do more useful work, probably with other nodes that the buggy one.  But that still leaves it a bug.

Reply:
I'm still not convinced this is acceptable...

Reply:
By which I mean, that this is a normal condition that is not a bug.

Reply:
Alright. I'll revert this to `error!`.